gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-account/src/test/java/io/gravitee/am/gateway/handler/account/services/AccountServiceTest.java

```diff
@@ -67,6 +80,12 @@ public class AccountServiceTest {
     @Mock
     private Domain domain;
 
+    @Mock
+    private UserService gatewayUserService;
+
+    @Mock
+    private PasswordService passwordService;
```

Comment:
unused variable

Reply:
It is not used but we need it to inject a non null PasswordService otherwise some tests are failing due NullPointerException